includes/api-management-multi-region-concepts.md

```diff
@@ -7,12 +7,17 @@ ms.author: danlep
 ---
 
 
-* With multi-region deployment, only the [gateway component](../articles/api-management/api-management-key-concepts.md#api-management-components) of your API Management instance is replicated to multiple regions. The instance's management plane and developer portal remain hosted only in the *primary* region, the region where you originally deployed the service. 
+* With multi-region deployment, only the [gateway component](../articles/api-management/api-management-key-concepts.md#api-management-components) of your API Management instance is replicated to multiple regions. The instance's management plane and developer portal remain hosted only in the *primary* region, the region where you originally deployed the service.
+  
+* If you want to configure secondary location for your APIM, the VNET and subnet region should match with the secondary location you are configuring. If you're adding/removing/enabling Availability zone in Primary region, or changing subnet of primary region, then VIP of APIM will change. Refer IP addresses of Azure API Management service (https://learn.microsoft.com/en-us/azure/api-management/api-management-howto-ip-addresses#changes-to-the-ip-addresses).
```

Comment:
```suggestion
* If you want to configure a secondary location for your instance that's deployed in a virtual network, the VNet and subnet region should match the secondary location you are configuring. 

* If you're adding, removing, or enabling availability zones in the primary region, or changing subnet of primary region, then the VIP address of your instance will change. See [IP addresses of Azure API Management service](api-management-howto-ip-addresses.md#changes-to-the-ip-addresses). However, if you're adding a secondary region, the primary region's VIP will not change, since every region has its own private VIP.
```